manifests/g/GoLang/Go/1.18.2/GoLang.Go.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: GoLang.Go
+PackageVersion: 1.18.2
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.google.com/go/go1.18.2.windows-amd64.msi
+  InstallerSha256: 3C6ECEA0966AEDAC4DD2E21A8DB5D67B984D979F69670CB4A6D767C5E0160296
+  ProductCode: '{4168B316-46AC-402D-ADE5-B14E57E298AA}'
```

```suggestion
  ProductCode: '{B6585533-BE54-4A40-87DD-987C404527E7}'
```
winget-create will solve this for you. Each installer has a unique ID.